src/common/config/base.json

```diff
@@ -11,7 +11,10 @@
     "showUnreadBadge": true,
     "useSpellChecker": true,
     "spellCheckerLocale": "en-US",
-    "helpLink": "https://docs.mattermost.com/help/apps/desktop-guide.html"
+    "helpLink": "https://docs.mattermost.com/help/apps/desktop-guide.html",
+    "enableMultiTeams": true,
+    "enableServerManagement": true,
+    "allowDarwinIconTray": true
```

Comment:
Probably unnecessary in this PR.

Reply:
@yuya-oc that config is related to this condition: https://github.com/mattermost/desktop/pull/600/files#diff-125da514cff642c8a155d6a3e3f118d0R403
I guess they when as part of the same commit. How would you like me to proceed? do another PR?

Reply:
@yuya-oc removed this logic and will add it to a new PR.